docs/eventing/README.md

```diff
@@ -208,26 +208,65 @@ The GitHubSource fires a new event for selected
 
 See the [GitHub Source](samples/github-source) example.
 
-### GcpPubSubSource
+### CloudPubSubSource
 
-The GcpPubSubSource fires a new event each time a message is published on a
+The CloudPubSubSource fires a new event each time a message is published on a
 [Google Cloud Platform PubSub topic](https://cloud.google.com/pubsub/).
 
 **Spec fields**:
 
-- `googleCloudProject`: `string` The GCP project ID that owns the topic.
 - `topic`: `string` The name of the PubSub topic.
-- `serviceAccountName`: `string` The name of the ServiceAccount used to access
-  the `gcpCredsSecret`.
-- `gcpCredsSecret`:
-  [ObjectReference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#objectreference-v1-core)
-  A reference to a Secret which contains a GCP refresh token for talking to
-  PubSub.
 - `sink`:
-  [ObjectReference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#objectreference-v1-core)
-  A reference to the object that should receive events.
+  [Destination](https://github.com/knative/pkg/blob/master/apis/duck/v1/destination.go)
+  A reference to the target that should receive events.
+
+See the [CloudPubSubSource](samples/cloudpubsubsource) example.
+
+### CloudStorageSource
+
+Registers for events of the specified types on the specified 
+[Google Cloud Storage](https://cloud.google.com/storage/) bucket and optional object prefix. 
+Brings those events into Knative.
+
+**Spec fields**:
+
+- `bucket`: `string` GCS bucket to subscribe to. For example my-test-bucket.
```

Comment:
can we add the `project` and `secret` to all of them... not the pubsub secret, because we will eventually remove it